src/app/Http/Controllers/UserProfileController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class UserProfileController extends Controller
+{
```

Comment:
add a docbloc like this

```suggestion
{
/**
 * Display the user profile creation form.
 *
 * @return \Illuminate\Contracts\View\View
 */
```